src/main/scala/strawman/collection/Iterator.scala

```diff
@@ -531,6 +531,50 @@ trait Iterator[+A] extends IterableOnce[A] { self =>
     // If *both* of them have no elements then the collections are the same
     hasNext == those.hasNext
   }
+
+  /** Returns this iterator with patched values.
+    * Patching at negative indices is the same as patching starting at 0.
+    * Patching at indices at or larger than the length of the original iterator appends the patch to the end.
+    * If more values are replaced than actually exist, the excess is ignored.
```

Comment:
@odd I copy-pasted the Scaladoc of the old collections but noticed that the first two points about patching at negative indices or larger than length are not respected by your implementation. I would prefer to stick to the same specification as the old collections. We can just copy-paste the old collections’ implementation, unless you see a reason to not do that?